task_2_Homework6.py

```diff
@@ -0,0 +1,39 @@
+﻿""" Создайте декоратор,
+    который вызывает задекорированную функцию
+    пока она не выполнится без исключений
+    (но не более n раз - параметр декоратора).
+    Если превышено количество попыток,
+    должно быть возбуждено исключение типа TooManyErrors
+
+"""
+import functools
+import random
+
+
+def retry(max_tries):
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            for n in range(1, max_tries + 1):
+                try:
+                    return func(*args, **kwargs)
+                except Exception:
+                    if n == max_tries:
+                        raise
```

Comment:
it should raise TooManyErrors exception according to task description and not depend on decorated function implementation